lib/index.js

```diff
@@ -771,6 +773,31 @@ class GalleryVideoArticle extends BaseAsset {
 
 exports.GalleryVideoArticle = GalleryVideoArticle;
 
+/**
+ * A simple wrapper over VideoAsset that should be used for toplevel videos.
+ *
+ * Along with the usual VideoAsset methods, it also supports set_author,
+ * which allows it to be used in place of GalleryVideoArticle.
```

Comment:
editorial nitpick: We never actually exposed GalleryVideoArticle in the API docs, so best not to mention it here. Try to give a bit more context about what this type of asset is for. For example, something like "This kind of asset is used for videos that aren't embedded as part of other content, where the video itself is the main item."

Reply:
I think I would prefer to keep the mention here so that the migration path is made clear. But I'll add the context.